index.js

```diff
@@ -269,6 +285,23 @@ class Main {
     }
   }
 
+  updateLambdaTableRows() {
+    const lambdaFunctionsWithDeploymentIndicator = JSON.parse(JSON.stringify(this.table.data));
```

Comment:
Why are stringify and parsing?

Reply:
it is used to deepClone the array of arrays